specs/protocol/isthmus/system-config.md

```diff
@@ -0,0 +1,29 @@
+# System Config
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+  - [`Roles`](#roles)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+The `SystemConfig` is updated to set a new role: the `OPERATOR_FEE_MANAGER`. This role is set once upon
+initialization and can only be set again with another call to `initialize`.
+
+### `Roles`
+
+The `Roles` struct is updated to include the new `OPERATOR_FEE_MANAGER` role.
```

Comment:
@tynes About the possible conflicts in `SystemConfig`, my concern is mainly about that part with the `Roles` struct that don't exist yet, but added in https://github.com/ethereum-optimism/optimism/pull/12932